crates/metal-codegen-llvm/src/stmt/function_definition.rs

```diff
@@ -43,6 +44,13 @@ impl CodeGenValue for FunctionDefinition<'_> {
 
             llvm.locals.clear();
 
+            for (idx, (input_name, _)) in self.signature.inputs.iter().enumerate() {
```

Comment:
Can .inputs be consumed here using .into_iter so that the input_name.to_string() call can be avoided a few lines below?